src/components/ClaimBookForm/ClaimBookForm.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react"
+import "./ClaimBookForm.css"
+
+function ClaimBookForm({id, bookclaim}) {
+    
+    const [name, setName] = useState('')
+    const [email, setEmail] = useState('')

```

Comment:
It looks like the state here isn't being used when sending the form, was there a reason for this?